.forge-snapshots/simple swap native.snap

```diff
@@ -1 +0,0 @@
-132772
```

Comment:
why is this one going?

Reply:
I ran rm -rf .forge-snapshots and then re-ran snapshots so it probably just didn't get deleted from a different PR

Reply:
seems like it is called "simple swap with native" now